languages/ruby/lib/oso.rb

```diff
@@ -2,11 +2,14 @@
 
 require 'oso/oso'
 require 'oso/polar'
+require 'oso/errors'
 require 'oso/version'
 
 # Top-level namespace for oso authorization library.
```

Comment:
```suggestion
# Top-level namespace for Oso authorization library.
```

Reply:
Heh good thing I'm touching all these files so we can find all the lowercase `oso`s -- will fix

We could maybe do a `s/oso/Oso/` and `s/polar/Polar` project-wide in one fell swoop soon too

Reply:
Yeah we should do that, but unfortunately there are probably plenty of perfectly fine occurrences of lowercase `oso` in the project, like `const oso = new Oso();`-style code snippets